<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetTCPServer</class>
 <widget class="QWidget" name="CWidgetTCPServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditSeverIP"/>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButtonSend">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextEdit" name="textEditRecv"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>服务器IP：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>服务器端口：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>客户端端口：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonBound">
     <property name="text">
      <string>绑定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditSeverPort">
     <property name="text">
      <string>10086</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>客户端IP：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEditClientPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="4">
    <widget class="QTextEdit" name="textEditSend"/>
   </item>
   <item row="0" column="4" colspan="3">
    <widget class="QComboBox" name="comboBoxClientIP">
     <item>
      <property name="text">
       <string>广播</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
